docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3.3"
 services:
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - 5432:5432
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
```

Comment:
Replace this line with `- ./database/postgres:/var/lib/postgresql/data`

Also add `database/postgres` to the `.gitignore` file at the root of the project please.